backend/src/apiserver/config/sample_config.json

```diff
@@ -25,7 +25,7 @@
     "file": "/samples/core/exit_handler/exit_handler.py.tar.gz"
   },
   {
-    "name": "[Sample] ML - TFX - Taxi Tip Prediction Model Trainer",
+    "name": "[Sample] Unified DSL - Taxi Tip Prediction Model Trainer",
```

Comment:
do you mind move this next to xgboost sample ? the samples are loaded in reverse order and would be nice basic samples are at top.

Reply:
Sure. Thanks for the fixing!